articles/users/guides/email-verified.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Email Verified Usage
+description: Outlines proper usage of the email_verified field in a user profile.
+topics:
+    - users
+    - user-management
+    - user-profiles
+contentType:
+    - how-to
+    - reference
+useCase: manage-users
+---
+# Verified Email Usage
+
+The `email_verified` field of a user profile indicates if the user has verified their email address. Valid emails are required for things such as communications to users, password resets/recovery, and passwordless magic links.
+
+An email is usually verified immediately after the user account is created or when the user logs in to the application for the first time. It's a good way to know that the person signing up actually owns the email at that moment. 
+
+Since email verification happens at that specific moment, later on we can't ensure the person logged in with the account still owns the email address. 
```

Comment:
```suggestion
Since email verification happens once at that specific moment, we can't ensure that a person who logs in with the user account at a later time still owns the email address that was verified. 
```